### PR TITLE
Quad fix: Symfony 3 compatibility, PHPUnit version lock, ACL option, Doc fix

### PR DESCRIPTION
**Pull request by zsturgess**
I know fixes should be their own PRs really, but these 3 are so small I couldn't resist batching them up.
Thoughts are explained on each commit message. Also, commits now signed :tada:

### CODE REVIEW

File: src/Resources/config/services.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 services:
     partnermarketing_file_system.factory:
-        class: "Partnermarketing\FileSystemBundle\Factory\FileSystemFactory"
+        class: "Partnermarketing\\FileSystemBundle\\Factory\\FileSystemFactory"
         arguments:
-            - %partnermarketing_file_system.default_file_system%
-            - %partnermarketing_file_system.config%
-            - %partnermarketing_file_system.tmp_dir%
+            - "%partnermarketing_file_system.default_file_system%"
```
Comment (EmilyShepherd) [inline src/Resources/config/services.yml]: These aren't strictly required are they?

File: src/Resources/config/services.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 services:
     partnermarketing_file_system.factory:
-        class: "Partnermarketing\FileSystemBundle\Factory\FileSystemFactory"
+        class: "Partnermarketing\\FileSystemBundle\\Factory\\FileSystemFactory"
```
Comment (EmilyShepherd) [inline src/Resources/config/services.yml]: Was this not working before? If it was I don't think we should double up on backslashes without reason

Comment (zsturgess) [inline src/Resources/config/services.yml]: Support for quoted single backslashes for FQCNs was deprecated in 2.8, removed in 3.0. If the double backslashes are abhorent to you, we can also remove the quotes and keep a single backslash, but anecdotally I've seen this pattern be more popular.

File: src/Resources/config/services.yml (hunk 1 above)
Comment (zsturgess) [inline src/Resources/config/services.yml]: Unquoted scalar values starting with '%' for parameters was deprecated in 3.1, to be removed in 4.0. You are correct that they're not strictly required right now, but whilst I'm here, might as well clean up deprecation warnings.

File: src/Resources/config/services.yml (hunk 2 above)
Comment (zsturgess) [inline src/Resources/config/services.yml]: (So yes, this file was causing Exceptions on Symfony >= 3.0)

Comment (EmilyShepherd) [inline src/Resources/config/services.yml]: Fair 'nuff :smile:

**Pull request merged**